motionplan/motionPlanner.go

```diff
@@ -99,6 +99,43 @@ func (req *PlanRequest) validatePlanRequest() error {
 
 // PlanMotion plans a motion from a provided plan request.
 func PlanMotion(ctx context.Context, request *PlanRequest) (Plan, error) {
+	// Calls Replan but without a seed plan
+	return Replan(ctx, request, nil, 0)
+}
+
+// PlanFrameMotion plans a motion to destination for a given frame with no frame system. It will create a new FS just for the plan.
+// WorldState is not supported in the absence of a real frame system.
+func PlanFrameMotion(ctx context.Context,
```

Comment:
how would you feel about putting everything other than the context & logger into a struct? This is a lot of parameters for a function.

Reply:
A PlanRequest doesn't quite fit here, because it requires a FrameSystem, and a goal as a PoseInFrame. Absent a FrameSystem, a PoseInFrame doesn't make sense, since there is no frame for the pose to be in.